composer.json

```diff
@@ -58,7 +61,7 @@
             "phpunit"
         ],
         "phpstan": [
-            "phpstan analyse"
+            "phpstan analyse --memory-limit=256M"
```

Comment:
why?

Reply:
I've removed this line, was for debugging purposes on my local machine

Reply:
thanks